package.json

```diff
@@ -41,15 +41,15 @@
     "start": "node server.js",
     "status": "git status",
     "test": "npm-run-all lint build test:unit test:es5 test:docker",
-    "test:docker": "chmod +x ./scripts/circleci-run-tests.sh && ./scripts/circleci-run-tests.sh",
+    "test:docker": "chmod +x ./scripts/run-docker-tests.sh && ./scripts/run-docker-tests.sh",
     "test:es5": "es-check es5 \"./es5/**/*.js\" ./dist/*.js",
     "test:framework:no-framework": "mocha tests/framework/no-framework.js",
     "test:framework:react:install": "cd ./tests/framework/react && rimraf ./node_modules package-lock.json && npm install",
     "test:framework:react:build": "cd ./tests/framework/react && npm run build",
     "test:framework:react:run": "mocha ./tests/framework/react.js",
     "test:framework:react": "npm-run-all test:framework:react:*",
     "test:frameworks": "npm-run-all test:framework:no-framework test:framework:react",
-    "test:integration": "karma start $PWD/karma.conf.ts --log-level debug",
+    "test:integration": "chmod +x ./scripts/run-integration-tests.sh && ./scripts/run-integration-tests.sh",
```

Comment:
Hmm. I have some opinions on this. Let me know if you disagree.

This command should only be to start integration tests. With this change, it will start integration tests and then report to slack. This hidden functionality doesn't seem like a good idea. What if by some very improbable chance that slack webhook environment variable is defined on someone's local machine?

Reply:
We don't report to slack if the env variables are not set. But I see your concern. I can change it.